src/OpenSSL/SSL.py

```diff
@@ -1507,6 +1507,16 @@ def get_cert_store(self):
         pystore._store = store
         return pystore
 
+    def set_cert_store(self, store):
```

Comment:
Can we added type annotations? Seems like we should start adding them as we go.

Reply:
+1 for type annotations :)